src/style.css

```diff
@@ -1,3 +1,69 @@
+* {
+  margin: 0;
+  padding: 0;
+  }
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  overflow-x: hidden;
+  width: 100%;
+
+}
+
+.header {
+  background-color: #FFF;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-align: center;
+  padding: 0 50px;
+  font-size: 12px;
+  font-weight: 500;
+
+
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  height: 40px;
+
+}
+
+.nav_list {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  list-style: none;
+  height: 60px;
+
+  /* padding-right: 50px; */
+}
+
+.nav_link {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+  position: relative;
+}
+
+.nav_link:hover {
+  color: #00ACDC;
```

Comment:
pls move reusable colors and sizes to variables

Reply:
thanks
add variables